### PR TITLE
Update docs and add state for junos_lag_interfaces

### PR DESCRIPTION
**Pull request by danielmellado**
This commit adds gathered state for junos_lag_interfaces and updates the
documentation to the new resource modules versioning for collections.

### CODE REVIEW

**Comment by pabelanger** [comment]
recheck

File: plugins/modules/junos_lag_interfaces.py [hunk 1]
```diff
@@ -30,14 +30,13 @@
...
-
-DOCUMENTATION = """module: junos_lag_interfaces
-short_description: Manage Link Aggregation on Juniper JUNOS devices.
+ANSIBLE_METADATA = {"metadata_version": "1.1", "supported_by": "network"}
+
+DOCUMENTATION = """
+---
+module: junos_lag_interfaces
+version_added: "1.0.0"
+short_description: Link Aggregation Juniper JUNOS resource module.
```
Comment (samccann) [inline plugins/modules/junos_lag_interfaces.py:39]:
```suggestion
short_description: Link Aggregation Juniper JUNOS resource module
```

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by justjais** [comment]
@danielmellado integration tests wrt to newly added states need to be added and you can also update the empty_config test as well for rendered and parsed

**Comment by justjais** [comment]
ignore the previous comment

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by pabelanger** [comment]
Until we land job redesign, I am going to close this issue (and reopen) to free up some capacity.

**Pull request merged**